app/code/Magento/Swatches/view/adminhtml/templates/catalog/product/attribute/visual.phtml

```diff
@@ -5,6 +5,7 @@
  */
 
 // phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
+// phpcs:disable Generic.Files.LineLength.TooLong
```

Comment:
I think Instead of disabling this check, we can resolve the line-length issue in this file. Please suggest.

Reply:
I disabled this check because resolving the line length issue would negatively impact the HTML formatting.